docker/lndmanage/Dockerfile

```diff
@@ -39,7 +43,7 @@ ARG LNDMANAGE_HOST_SRC_PATH
 COPY "$LNDMANAGE_HOST_SRC_PATH" .
 
 # this should be fast because it is cached from builder image
-RUN pip install -r requirements.txt
```

Comment:
I think we should be able to remove that whole install instruction, since we run `./lndmanage.py` directly, without installing into the virtual environment and therefore we don't create an alias `lndmanage`, see `run-lndmanage`.